callout/src/main/java/com/google/apigee/JavaCalloutRemotePolicyExecution.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.apigee;
+
+import com.apigee.flow.execution.ExecutionContext;
+import com.apigee.flow.execution.ExecutionResult;
+import com.apigee.flow.execution.spi.Execution;
+import com.apigee.flow.message.MessageContext;
+import com.google.apigee.Execute.ExecutionResult.Action;
+import com.google.apigee.ProtoMessageBuilders.ExecutionContextProtoMessageBuilder;
+import com.google.apigee.ProtoMessageBuilders.MessageContextProtoMessageBuilder;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * Java Callout demonstrating execution of a Java Callout or Apigee Policy on remote HTTP Server.
```

Comment:
could we make a comment that this assumes that java callout or apigee policy makes a change to the specific flow variable? I don't know if we plan on making this class more generalized, but we could leave this as a TODO for the future